source/sass/component/_footer.scss

```diff
@@ -56,21 +72,128 @@ $c-footer-logotype-height: calc(var(--c-footer-logotype-height, 6) * #{$base}) !
         margin-left: $base;
         margin-right: $base;
     }
+    
+    &__widget-area {
+        gap: calc(#{$base} * 2);
+        .footer-title {
+            margin-bottom: calc(#{$base} * 2);
+        }
+        ul, p, .wp-block-image {
```

Comment:
It feels kind of odd to have a wordpress specific thing in here, is it possible to style it without using a wordpress class?